.github/dependabot.yml

```diff
@@ -9,6 +9,8 @@ updates:
       - "/site"
     schedule:
       interval: "weekly"
+    allow:
+      - dependency-type: "indirect"
```

Comment:
@kannon92 do you know about any risks of enabling this, like a lot of churn of PRs which aren't CVE fixes? Or maybe some PRs would be invalid? Do you know projects enabling this? 

I'm ok with an experiment, but would like to get the answers and also consult with @tenzen-y

Reply:
Instead of this change, I would recommend enabling dependabot Security notification: https://github.com/kubernetes-sigs/kueue/security/dependabot

This allows us to upgrade dependencies regardless of direct and indirect.

Reply:
+1 to investigate as an option

Reply:
I enabled the dependabot security features.
<img width="836" alt="Screenshot 2025-04-16 at 18 07 16" src="https://github.com/user-attachments/assets/f862287c-7a0d-4dc5-b39f-213c7199f8b5" />